gluten-substrait/src/main/scala/org/apache/gluten/extension/columnar/CommonSubexpressionEliminateRule.scala

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.gluten.extension
+package org.apache.gluten.extension.columnar
```

Comment:
(minor) This is an optimizer rule rather than a columnar rule so ideally we can keep it in package `org.apache.gluten.extension`